ui/libproject/devstart.rb

```diff
@@ -0,0 +1,94 @@
+require "optparse"
+require_relative "utils/common"
+
+def install_dependencies()
+  common = Common.new
+  common.docker.requires_docker
+
+  parent = Dir.chdir("..") { Dir.pwd }
+
+  common.run_inline %W{docker run --rm -w /w/ui -v #{parent}:/w:cached node npm install}
+end
+
+def swagger_regen()
+  common = Common.new
```

Comment:
If you call `./project.rb swagger-regen`, your UI will automatically reload since they share the code via the host's file system. Seem okay?